docs/topics/data-analysis/data-analysis-connect-to-db.md

```diff
@@ -0,0 +1,185 @@
+[//]: # (title: Connect and retrieve data from databases)
+
+Kotlin Notebook offers capabilities for connecting to and retrieving data from various types of SQL databases, such as 
+MariaDB, PostgreSQL, MySQL, and SQLite. 
+Utilizing the [Kotlin DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html), Kotlin Notebook can establish 
+connections to databases, execute SQL queries, and import the results for further operations.
+
+For a detailed example, see the [Notebook in the KotlinDataFrame SQL Examples GitHub repository](https://github.com/zaleslaw/KotlinDataFrame-SQL-Examples/blob/master/notebooks/imdb.ipynb).
+
+## Before you start
+
+1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/?section=mac).
+2. Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook) in IntelliJ IDEA.
+
+   > Alternatively, access the Kotlin Notebook plugin from **Settings/Preferences** | **Plugins** | **Marketplace** within IntelliJ IDEA.
+   >
+   {type="tip"}
+
+3. Create a new Kotlin Notebook by selecting **File** | **New** | **Kotlin Notebook**.
+4. Ensure you have access to an SQL database, such as MariaDB or MySQL.
+
+
+## Connect to database
+
+You can connect to and interact with an SQL database using specific functions from the [Kotlin DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html). 
+You can use `DatabaseConfiguration` to establish a connection to your database and `getSchemaForAllSqlTables()` to retrieve 
+the schema of all tables within it.
+
+Let's look at an example:
```

Comment:
Here and in the list in the second section, code blocks are not aligned with list items